themes/bootstrap3/js/openurl.js

```diff
@@ -58,12 +58,13 @@ VuFind.register('openurl', function OpenUrl() {
       return false;
     });
 
-    if (typeof Hunt === 'undefined' || VuFind.isPrinting()) {
+    if (VuFind.isPrinting()) {
       container.find('.openUrlEmbed.openUrlEmbedAutoLoad a').trigger('click');
     } else {
-      new Hunt(
-        container.find('.openUrlEmbed.openUrlEmbedAutoLoad a').toArray(),
-        { enter: embedOpenUrlLinks }
+      VuFind.observerManager.createIntersectionObserver(
+        'openUrlEmbed',
+        embedOpenUrlLinks,
+        Array.from(container.querySelectorAll('.openUrlEmbed.openUrlEmbedAutoLoad a'))
```

Comment:
The container is transformed into a jquery object in line 46.

Reply:
It was a mixed case. I made it all jQuery now, for minimal diffs.